src/api/app/controllers/comments_controller.rb

```diff
@@ -32,7 +32,7 @@ def find_obj
       @obj = User.find_by!(login: params[:user_login])
       @header = { user: @obj.login }
     else
-      @obj = BsRequest.find_by_number!(params[:id])
+      @obj = BsRequest.find_by_number!(params[:request_number])
```

Comment:
That controller is not in the webui namespace. Is it also exposed to the API?

In that case we would have to keep the old parameter functional. At least for now. We could deprecate it and later drop it.